### PR TITLE
Added missing options to FTP adapter

### PR DESCRIPTION
**Pull request by Radiergummi**
Solves #87.

### CODE REVIEW

File: src/Adapter/Builder/FtpAdapterDefinitionBuilder.php [hunk 1]
```diff
@@ -66,12 +66,36 @@ protected function configureOptions(OptionsResolver $resolver)
...
+
+        $resolver->setDefault('recurse_manually', true);
+        $resolver->setAllowedTypes('recurse_manually', 'bool');        
     }
 
     protected function configureDefinition(Definition $definition, array $options)
     {
+        $options['transferMode'] = $options['transfer_mode'];
+        $options['systemType'] = $options['system_type'];
+        $options['timestampsOnUnixListingsEnabled'] = $options['timestamps_on_unix_listings_enabled'];
```
Comment (tgalopin) [inline src/Adapter/Builder/FtpAdapterDefinitionBuilder.php:89]: Could we perhaps use the same name as the Flysystem option (ie in camel case by default)? If would avoid having to remap like this and would ease discoverability I think.

Comment (Radiergummi) [reply src/Adapter/Builder/FtpAdapterDefinitionBuilder.php:89]: I'd favor that too, but went with the existing code (when in Rome, do as the Romans do 🙂)

Comment (tgalopin) [reply src/Adapter/Builder/FtpAdapterDefinitionBuilder.php:89]: Okay, it's as you prefer then, I don't mind either way.

Tests are failing though :)

Comment (Radiergummi) [reply src/Adapter/Builder/FtpAdapterDefinitionBuilder.php:89]: Shouldn't, anymore. Could you approve again please?

**Review by maxhelias — APPROVED** [review]

**Comment by tgalopin** [comment]
Thanks!

**Comment by maxhelias** [comment]
@wujku I can't, I'm not part of this organization

**Comment by wujku** [comment]
@maxhelias Sorry, I missed that @tgalopin merged this changes.
@tgalopin maybe v2.2.1 with this modification?

**Comment by tgalopin** [comment]
Released: https://github.com/thephpleague/flysystem-bundle/releases/tag/2.2.1

**Pull request merged**